misc/chrome_extension/js/panel.js

```diff
@@ -641,7 +655,7 @@
 		ruleElement.childNodes[1].childNodes[5].value = queryToAdd;
```

Comment:
Please use a variable for `ruleElement.childNodes[1]`, it's repeated a bunch of times here, and it will help explaining what it is.